cpp/ql/lib/semmle/code/cpp/stmts/Stmt.qll

```diff
@@ -213,6 +213,26 @@ class ConditionalStmt extends ControlStructure, TConditionalStmt { }
 class IfStmt extends ConditionalStmt, @stmt_if {
   override string getAPrimaryQlClass() { result = "IfStmt" }
 
+  /**
+   * Gets the initialization statement of this 'if' statement.
+   *
+   * For example, for
+   * ```
+   * if (x = y; b) { f(); }
```

Comment:
Should we also have an example with a declaration like this?
```cpp
if(int x = y; b) { f(); }
```

Reply:
Tempted to make that the only example, as that's the common use-case, while `if (x = y; b)` is a bit esoteric. What do you think?

Reply:
Yeah, I think that would be fine as well.

Reply:
Done.